packages/components/src/globals/scss/_css--reset.scss

```diff
@@ -125,12 +125,7 @@
     // Chrome 62 fix
     button,
     select,
-    input[type='button'],
-    input[type='submit'],
-    input[type='reset'],
-    input[type='file'],
-    input[type='text'],
-    input[type='password'],
+    input,
```

Comment:
@emyarod Is this intentional? Asking because it seems to change the specificity of the reset style for `<input>`.

Reply:
@asudoh yes, in order to override browser user agent styles

Reply:
I see, is it needed to complete this PR or was it just an issue you found?

Reply:
OK sounds fair enough - Thanks for clarifying!